### PR TITLE
Feature/softmax speed up

### PR DESCRIPTION
**Pull request by dcaustin33**
Computing softmax on GPU vs CPU, eliminates .4 seconds per step. Also a quick code cleanup of some functions we are no longer using

### CODE REVIEW

File: dataquality/integrations/torch_semantic_segmentation.py [hunk 1]
```diff
@@ -295,14 +295,16 @@ def calculate_mislabeled_pixels(
         mislabeled_pixels = mislabeled_pixels[-bs:]
         return mislabeled_pixels
 
-    def get_argmax_and_logits(self) -> Tuple[torch.Tensor, torch.Tensor]:
+    def get_argmax_probs(
+        self,
+    ) -> Tuple[torch.Tensor, torch.Tensor]:
         """Helper function to get the argmax and logits from the model outputs
```
Comment (elboy3) [inline dataquality/integrations/torch_semantic_segmentation.py:301]: docstring still says logits

**Review by elboy3 — APPROVED** [review]
